### PR TITLE
Adjust the information shown in the floating window, clean up the code

### PR DESCRIPTION
**Pull request by chenj47**
Adding comments and adjust the information displayed for floating window

### CODE REVIEW

File: js/bars.js [hunk 1]
```diff
@@ -1027,9 +1042,28 @@ historian.Bars.prototype.filter_ = function(data, callback) {
...
   return this.filter_(data, function(entry) {
-    var value = entry.value;
-    var valueMap = value.split(",");
-    var eventUID = valueMap[6];
+    // Only keep the entries that corresponds to the given uid.
+    var valueMap = entry.value.split(",");
+    var eventUID;
+    switch(valueMap[0]) {
+      // For error entry, the value field starts with an error tag, so we
+      // case on the error tag to get the uid.
```
Comment (agarwal-v) [inline js/bars.js]: switch on the error tag instead of case

File: js/bars.js [hunk 2]
```diff
@@ -1393,33 +1429,24 @@ historian.Bars.prototype.createSensorErrorTable_ = function(values) {
...
   ];
 
-  var bodyRows = values.map(function(entry) {
+  var bodyRows = [];
+  values.forEach(function(entry) {
     var v = entry.value.split(',');
     var uid, packageName, errorMsg;
     // Obtain the uid and package name from the error message, it is possible
-    // that some error message does not contain ui and package name information.
+    // that some error message does not contain uid or package name information.
```
Comment (agarwal-v) [inline js/bars.js]: that an error message.....

File: js/bars.js [hunk 3]
```diff
@@ -1458,35 +1495,85 @@ historian.Bars.prototype.createSensorTable_ = function(values) {
...
+    } else if (isActive) {
+      singleRow = [startTime, endTime, v[6], v[7], samplingRate, batching, 
+        durationOutput,actualSamplingRate, actualBatchingPeriod];
+    } else {
+      singleRow = [startTime, endTime, v[6], v[7], samplingRate, batching, 
+        durationOutput];
+    }
+    // It is possible that multiple events with the same values are created, 
+    // then the entry.count field will record the number of events having this
+    // entry value.We want to show all events even though the values
```
Comment (agarwal-v) [inline js/bars.js]: Space after period.

Comment (agarwal-v) [reply js/bars.js]: If possible about 1st person pronouns, I, We, etc

You can say "Show all events ..."

File: js/bars.js [hunk 4]
```diff
@@ -1458,35 +1495,85 @@ historian.Bars.prototype.createSensorTable_ = function(values) {
...
-    return headRow.length > 6 ? 
-    [v[0], v[2], v[6], v[7], samplingRate, batching, v[10], durationOutput] :
-    [v[0], v[2], v[6], v[7], v[10], durationOutput];
+    var isActive = false;
+    if (v[v.length - 1] == "isActiveConn") {
+      highlightActiveConn.push(index);
+      isActive = true;
+      endTime = "Running";
+      durationOutput = "Active Connection";
+      // For active connection, we also show the actual sampling rate and
```
Comment (agarwal-v) [inline js/bars.js]: remove we

**Review by agarwal-v — COMMENTED** [review]
Minor comment changes requested.

File: js/bars.js (hunk 1 above)
Comment (chenj47) [reply js/bars.js]: I am not sure what you mean here. Should I just use if conditions instead of switch cases?

Comment (agarwal-v) [reply js/bars.js]: In the comment, replace case with switch.

File: js/bars.js [hunk 5]
```diff
@@ -827,13 +837,16 @@ historian.Bars.prototype.renderSeries_ = function(data) {
       var showBars = this.container_.find(SHOW_BARS_TOGGLE_).is(':checked');
       // Access the pattern embedded in this svg.
       var hatchPattern = '#' + this.container_.find('svg pattern').attr('id');
-
+
```
Comment (agarwal-v) [inline js/bars.js]: remove blank line

Comment (agarwal-v) [reply js/bars.js]: or remove spaces and keep the line

File: js/bars.js [hunk 6]
```diff
@@ -1862,8 +1956,8 @@ historian.Bars.prototype.createSortedTable_ = function(series, cluster) {
  */
 historian.Bars.prototype.createBroadcastsTable_ = function(series, cluster) {
   // The current series either contains all enqueue events, or all dispatch
-  // events. We also want to show details from the corresponding series.
-  // e.g. for enqueue foreground events, we want to show the corresponding
+  // events. Show details from the corresponding series .
```
Comment (agarwal-v) [inline js/bars.js]: no space before. series . -> series.

File: js/bars.js [hunk 7]
```diff
@@ -1937,8 +2031,8 @@ historian.Bars.prototype.createBroadcastsTable_ = function(series, cluster) {
       // screen anyway.
       .filter(function(aggId, idx) { return idx < 100; });
   var bodyRows = originalEntryIds.map(function(aggId) {
-    // Broadcast events may be overlapping and sliced up, but we want
-    // to show the enqueue / dispatch times based on the original entries.
+    // Broadcast events may be overlapping and sliced up, but the
```
Comment (agarwal-v) [inline js/bars.js]: remove trailing spaces

File: js/bars.js [hunk 8]
```diff
@@ -2049,28 +2143,46 @@ getColorScale = function(number) {
...
- * Returns the intensity of the given sampling rate information
+/**	
+ * Returns the intensity of the given sampling rate information	
  * @param {number} requestMode Request mode of the sensor.
- * @param {number} sensorMaxRate The max sampling rate set for the sensor.
- * @param {number} curMaxRate The max sampling rate seen in the interval.
- * @return {object} 
+ * @param {number} curMaxRate The max sampling rate seen in the interval.		
+ * @param {number} sensorMaxRate The max sampling rate set for the sensor.	
+ * @return {object}
```
Comment (agarwal-v) [inline js/bars.js]: trailing spaces in the comments

File: js/bars.js [hunk 9]
```diff
@@ -243,14 +243,14 @@ historian.Bars.prototype.createOnDragHandler_ = function(bars) {
...
-      // group being dragged, we need to add a y-translate.
+      // group being dragged, a y-translate is needed.
       if (group.index >= min && group.index <= max) {
         if (d.index == group.index) {
           // This is the group being dragged.
           yTranslate = draggedTranslate;
         } else {
-          // We need to move this group one row in the opposite direction
-          // of the group being dragged.
+          // Move this group one row in the opposite direction of the group
```
Comment (agarwal-v) [inline js/bars.js]: trailing space

**Review by agarwal-v — COMMENTED** [review]
lots of trailing spaces got introduced in this patch

File: js/bars.js [hunk 10]
```diff
@@ -2049,28 +2143,46 @@ getColorScale = function(number) {
   return colorScale;
 };
 
-/**
- * Returns the intensity of the given sampling rate information
+/**
```
Comment (agarwal-v) [inline js/bars.js]: trailing spaces...

File: js/bars.js [hunk 11]
```diff
@@ -2049,28 +2143,46 @@ getColorScale = function(number) {
   return colorScale;
 };
 
-/**
- * Returns the intensity of the given sampling rate information
+/**	
+ * Returns the intensity of the given sampling rate information
```
Comment (agarwal-v) [inline js/bars.js]: trailing space

File: js/bars.js (hunk 7 above)
Comment (agarwal-v) [reply js/bars.js]: Still there

**Review by agarwal-v — COMMENTED** [review]
Still a few more spaces!!

File: js/bars.js [hunk 12]
```diff
@@ -1259,13 +1301,13 @@ historian.Bars.prototype.tooltipText_ = function(
     }
   } else {
     // Attach a div that the table can be appended to.
-    // The table shouldn't be appended here as we may just want the text
-    // output for copying the tooltip.
+    // The table shouldn't be appended here as only the text output for
```
Comment (agarwal-v) [inline js/bars.js]: One last one on line 1304

**Review by agarwal-v — COMMENTED** [review]
One last trailing space

File: js/bars.js (hunk 12 above)
Comment (chenj47) [reply js/bars.js]: Can you please check the lastest commit with comment "space", the one with ID 276c080?

**Review by agarwal-v — APPROVED** [review]

**Pull request merged**